hummingbot/client/config/conf_migration.py

```diff
@@ -28,6 +29,7 @@ def migrate_configs(secrets_manager: BaseSecretsManager) -> List[str]:
         migrate_strategy_confs_paths()
         errors.extend(migrate_connector_confs(secrets_manager))
         store_password_verification(secrets_manager)
+        migrate_xemm_confs()
```

Comment:
Should also add it after line 43 below in `migrate_strategies_only`, in case where we don't have any legacy secure configs to migrate. Or better yet, add it to `migrate_strategy_confs_paths` and rename it to `migrate_strategy_confs` since we won't be only changing paths anymore

Reply:
👇